docs/source/commands/peerlifecycle.md

```diff
@@ -441,7 +441,10 @@ channel members have approved a chaincode definition to meet the
 ``Application/Channel/Endorsement`` policy (a majority by default) before the
 definition can be committed to a channel.
 
-     * ```
+  * Here is an example of the `peer lifecycle chaincode checkcommitreadiness` command,
+    which checks a chaincode named `mycc` at version `1.0` on channel `mychannel`.
+
+    ```
     export ORDERER_CA=/opt/gopath/src/github.com/hyperledger/fabric/peer/crypto/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
     .
```

Comment:
i believe this dot is not necessary

Reply:
Thank you for having reviews.
New patch removes unnecessary dots in the entire file.